src/dependency-manager/src/component-config/base.ts

```diff
@@ -10,7 +10,7 @@ export interface ParameterDefinitionSpec {
 }
 
 export abstract class ComponentConfig extends BaseSpec {
-  abstract __version: string;
+  abstract __version?: string;
```

Comment:
Why optional now? Or should it have always been that way?

Reply:
We don't use and it muddles the output when you convert to json.

Reply:
Could we just kill it then?

Reply:
We can, but trying to avoid another data migration. Any properties we remove would require it.